src/main/java/com/minecolonies/entity/ai/util/AIState.java

```diff
@@ -160,4 +162,27 @@
      */
     BUILDER_COMPLETE_BUILD,
 
+    /*
+###FARMER###
+    */
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)